### ISSUE TITLE
AQA testing cannot access web context on web auth with auth0.

### ISSUE THREAD

**Issue by aboiko19** [auth0/Auth0.swift#771]
### Checklist

- [X] I have looked into the [Readme](https://github.com/auth0/Auth0.swift#readme), [Examples](https://github.com/auth0/Auth0.swift/blob/master/EXAMPLES.md), and [FAQ](https://github.com/auth0/Auth0.swift/blob/master/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/Auth0.swift/documentation/auth0) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/Auth0.swift/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

Hello, hope for your help. 

In our application, we use AQA testing, everything worked fine before and we could pass the web auth.
But now getting web context is impossible for all devices with version 16.4. We investigate that from iOS 16.4, WKWebView needs to be made additionally inspectable, but if I understand correctly, the auth0 framework uses SFSafarViewController internally and these changes did not affect him.
Tell me if additional settings are required from our side for AQA. I've looked at every possible option, and at the moment the only one who can clarify the problem is you.

We use all the code that is indicated in the documentation. 
```
 Auth0
            .webAuth(clientId: client, domain: domain)
            .useEphemeralSession()
            .scope(AuthServiceConstants.auth0Scope)
            .audience(audience)
            .start { [weak self] result in
            // Application logic
}
```

### Describe the ideal solution

Be able to get web context which is available in SFSafariViewController as before

### Additional context

Devices below iOS version 16.4 work as before. 
The Auth0 library is installed using the Swift Package Manager, currently using version 2.3.2

**Comment by Widcket** [comment]
Hi @aboiko19, thanks for raising this.

This library uses `ASWebAuthenticationSession` and `SFSafariViewController` under the hood. `WKWebView` is not used. In your particular case, you're calling `.useEphemeralSession()`, so the SDK is using `ASWebAuthenticationSession`.

> Tell me if additional settings are required from our side for AQA. 

I'm not aware of any additional settings being necessary. Since this library merely wraps `ASWebAuthenticationSession` and `SFSafariViewController`, I'd suggest looking for any changes related to UI-testing `ASWebAuthenticationSession`.

**Comment by Widcket** [comment]
Closing, as I believe the above answers your question. Please feel free to ping if you'd prefer to reopen.

**Comment by aboiko19** [comment]
Hi @Widcket, thanks for your answer. 

I looked at the available code, and you confirmed everything. But I also found on the Apple portal that for some developers, ASWebAuthenticationSession's also ceased to be available in debug mode. It looks like it's on Apple's side, and we have to wait for their clarification. It's interesting that WKWebView is mentioned and more configuration is needed for them, but ASWebAuthenticationSessions will be required as well. 

In any case, thank you for your reply.

Best regards
